macros/edr/dbt_artifacts/upload_source_freshness_results.sql

```diff
@@ -0,0 +1,28 @@
+{% macro upload_source_freshness_results() %}
+  {% set source_freshness_results_relation = ref('elementary', 'source_freshness_results') %}
```

Comment:
cosmetics - just to follow our convention of dbt_ - wdyt about calling it dbt_source_freshness?

Reply:
You're comparing it to an artifact (`dbt_`), but it's a run result (`_results`).